### PR TITLE
[CDAP-6392] Updated exception thrown for namespace creation

### PR DESCRIPTION
**Pull request by knazir**
Issue: https://issues.cask.co/browse/CDAP-6392
Build: http://builds.cask.co/browse/CDAP-RUT89-5

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java [hunk 1]
```diff
@@ -191,17 +191,15 @@ private void validateCustomMapping(NamespaceMeta metadata) throws Exception {
       // if hbase namespace is provided validate no other existing namespace is mapped to it
       if (!Strings.isNullOrEmpty(metadata.getConfig().getHbaseNamespace()) &&
         metadata.getConfig().getHbaseNamespace().equals(existingConfig.getHbaseNamespace())) {
-        throw new NamespaceAlreadyExistsException(existingNamespaceMeta.getNamespaceId().toId(),
-                                                  String.format("A namespace '%s' already exists with the given " +
+        throw new IllegalArgumentException(String.format("A namespace '%s' already exists with the given " +
```
Comment (anwar6953) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java]: what status code does the REST API return due to this?
500? 400?
Can you try it out?

**Comment by anwar6953** [comment]
lgtm

**Comment by knazir** [comment]
Exception now returns a 400 response @anwar6953

**Comment by knazir** [comment]
Waiting on guild to be breen: http://builds.cask.co/browse/CDAP-RUT89-2

**Comment by knazir** [comment]
Waiting on guild to breen: http://builds.cask.co/browse/CDAP-RUT89-5

**Comment by knazir** [comment]
Guild is breen: http://builds.cask.co/browse/CDAP-RUT89-5 going to merge.

**Pull request merged**